paymentsheet/src/main/java/com/stripe/android/paymentsheet/Element.kt

```diff
@@ -113,6 +113,7 @@ enum class ElementType {
     SaveForFutureUse,
     Mandate,
     IdealBank,
+    GenericText
```

Comment:
Make it `SimpleText` too?

Reply:
I am going to be removing this in an upcoming PR, so if it is ok with you, I will skip this for now.

Reply:
https://github.com/stripe/stripe-android/pull/4023